strawberryfields/circuitspecs/chip0.py

```diff
@@ -19,6 +19,7 @@
 from scipy.linalg import block_diag
 
 from strawberryfields.program_utils import CircuitError, Command, group_operations
+from strawberryfields.parameters import par_evaluate
```

Comment:
`par_evaluate` is a function which evaluates symbolic parameters, but in a 'vectorized' fashion? So it works with single parameters, as well as sequences of parameters?

Reply:
Right.